src/jvm/io/fsq/twofishes/indexer/output/PrefixIndexer.scala

```diff
@@ -112,32 +112,36 @@ class PrefixIndexer(
       if (index % 1000 == 0) {
         log.info("done with %d of %d prefixes".format(index, numPrefixes))
       }
-      val records = getRecordsByPrefix(prefix, PrefixIndexer.MaxNamesToConsider)
+      try {
```

Comment:
honest question: would it make sense to have this wrap just the `getRecordsByPrefix` call instead of the entire block here? I suppose any exception leaves you in the same broken state, but I'm not sure you would want to ignore every possible error here